icons/icloud.svg

```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" role="img"><title>iCloud icon</title><path d="M20.013 10.726l.001-.028A6.346 6.346 0 0 0 8.09 7.67a3.414 3.414 0 0 0-4.989 2.829A4.72 4.72 0 0 0 0 14.929a4.72 4.72 0 0 0 4.719 4.719h14.807a4.475 4.475 0 0 0 .487-8.922z"/></svg>
\ No newline at end of file
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>iCloud icon</title><path d="M20.02272 10.7263l.001-.028a6.34908 6.34908 0 00-11.9298-3.02948 3.41566 3.41566 0 00-4.99142 2.83037A4.7223 4.7223 0 000 14.93134a4.7223 4.7223 0 004.7213 4.7213h14.81419a4.47717 4.47717 0 00.48723-8.92634z"/></svg>
```

Comment:
There seem to be some differences between this version (red) and [SVG source](https://upload.wikimedia.org/wikipedia/commons/1/1c/ICloud_logo.svg) (black). Any chance you could rebuild from source? Would also be great to get it to 3 decimals if possible!
![image](https://user-images.githubusercontent.com/1830867/103036857-9131be80-4562-11eb-9c76-ef0a0c146bb5.png)

Reply:
@adamrusted Done ☁️